src/migration/ember-addon/steps/create-options.js

```diff
@@ -89,13 +102,7 @@ function deriveAddonLocation(addonPackage) {
   }
 
   // eslint-disable-next-line no-unused-vars
-  const [scope, packageName] = addonPackage.name.split('/');
-
-  if (!packageName) {
-    throw new SyntaxError(
-      `ERROR: In package.json, the package name \`${addonPackage.name}\` is not valid.`
-    );
-  }
```

Comment:
In [`ember-codemod-pod-to-octane`](https://github.com/ijlee2/ember-codemod-pod-to-octane/pull/19/commits/e1b690078a15f32886f77d785cf20eff71300f42#diff-c6a1934d498db73b42744184a83f28d3b04080109a279f5fb9499f79d3a64b62R19-R26), I realized that it's not a good idea to perform validation inside this function (concerns weren't separated well).